ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__arm64-stable.yaml

```diff
@@ -67,6 +67,11 @@ releases:
       architecture: amd64
       channel: candidate
       version: "4.16"
+  source:
```

Comment:
stable ci does not run any test suite, so we do not need the setting here.

Reply:
remove it, thanks